.eslintrc.js

```diff
@@ -1,5 +1,3 @@
 module.exports = {
-  extends: [
-    './packages/eslint-config-nimble/lib/index.js'
-  ],
-}
+  extends: ['./packages/eslint-config-nimble-core/lib/index.js'],
```

Comment:
```suggestion
  extends: './packages/eslint-config-nimble-core/lib/index.js',
```

Does it work without the brackets? 🤔 (I see in other files you removed them that's the reason I suggest it here too 😇 )

Reply:
It does, updated in [1af4364](https://github.com/nimblehq/eslint-config-nimble/pull/49/commits/1af43648757c88b92f31a824419b206ce635d8fa)